config/baas_config.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+import "os"
+
+type BaasConfig struct {
+	BaasApiKey     string
+	BaasWebhookUrl string
+	BaasUrl        string
+	BaasMtlsKey    string
+	BaasMtlsCert   string
+}
+
+func getBaasConfig() BaasConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀 
```suggestion
func getBaasConfig() *StoneBaasConfig {
```